src/components/FooterPostLogin/FooterPostLogin.tsx

```diff
@@ -18,11 +18,13 @@ export const FooterPostLogin = ({
   ...langProps
 }: FooterPostLoginProps): JSX.Element => {
   const wrapHandleClick =
```

Comment:
put this function in a common place in order to reuse it.
/src/utils/ts-utils could be fine as destination for the function